community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -94,6 +94,9 @@
             "dbms.cypher.planner",
             options( "COST", "RULE", DEFAULT ), DEFAULT );
 
+    @Description( "Set this to specify the behavior for conformance to Cypher hints." )
```

Comment:
Perhaps this could be worded with a bit more detail? These strings show up in the manual and thus should be explicit and clear.